src/lib/list/list.scss

```diff
@@ -27,7 +28,18 @@ $mat-dense-three-line-height: 76px;
 $mat-dense-multi-line-padding: 16px;
 $mat-dense-list-icon-size: 20px;
 
-$mat-list-item-inset-divider-offset: 72px;
+// This mixin provides the correct offset for an inset divider based on the
+// size of the parent class (e.g. avatar vs icon)
+@mixin mat-inset-divider-offset($offset) {
```

Comment:
The mixin should live somewhere in the divider module since it could be used in other modules that consume it.

Reply:
Moved